ballerina/Module.md

```diff
@@ -2,6 +2,10 @@
 
 This module provides file read/write APIs and console print/read APIs. The file APIs allow read and write operations on different kinds of file types such as bytes, text, CSV, JSON, and XML. Further, these file APIs can be categorized as streaming and non-streaming APIs.
 
+The following diagram depicts the overview architecture of the I/O package.
+
+![Overview](docs/architecture.svg)
```

Comment:
Does this path work? In the bala, this file is located inside the `docs/<module-dir-path>` directory and the diagram is located in the `docs` directory.
I am not sure whether `bal doc` command will give us some idea here. May be we have to push it to staging and check

Can't we just give the github url for this image: https://raw.githubusercontent.com/ballerina-platform/module-ballerina-io/master/ballerina/resources/architecture.svg
I think in connectors we use the github links

Reply:
yes, I am planning to push this to the staging central and check whether it is working. Inside the bala file, it is inside the module,
<img width="447" alt="Screenshot 2024-08-06 at 18 15 46" src="https://github.com/user-attachments/assets/5a1a558b-bb91-4847-9c1f-b61deeb75177">

Reply:
Isn't it better to give the URL?

Reply:
Added in [e938e18](https://github.com/ballerina-platform/module-ballerina-io/pull/1223/commits/e938e18e1c8582fea6d853e23c7406d57107b16c)